package.json

```diff
@@ -1,1031 +1,1037 @@
+		"analytics-node": "^5.1.2",
+		"bson": "^4.6.1",
+		"bson-transpilers": "^1.7.0",
+		"classnames": "^2.3.1",
+		"debug": "^4.3.3",
+		"dotenv": "^8.6.0",
+		"micromatch": "^4.0.4",
+		"mongodb": "^4.4.1",
+		"mongodb-cloud-info": "^1.1.3",
+		"mongodb-connection-string-url": "^2.5.2",
+		"mongodb-data-service": "^21.18.0",
+		"mongodb-ns": "^2.3.0",
+		"mongodb-schema": "^9.0.0",
+		"numeral": "^2.0.6",
+		"react": "^16.14.0",
+		"react-dom": "^16.14.0",
+		"react-redux": "^7.2.6",
+		"redux": "^4.1.2",
+		"ts-log": "^2.2.4",
+		"uuid": "^8.3.2",
+		"vscode-languageclient": "^7.0.0",
+		"vscode-languageserver": "^7.0.0",
+		"vscode-languageserver-textdocument": "^1.0.4"
+	},
+	"devDependencies": {
+		"@types/analytics-node": "^3.1.7",
+		"@types/babel__core": "^7.1.18",
+		"@types/babel__traverse": "^7.14.2",
+		"@types/chai": "^4.3.0",
+		"@types/chai-fs": "^2.0.2",
+		"@types/chai-json-schema": "^1.4.6",
+		"@types/debug": "^4.1.7",
+		"@types/enzyme": "^3.10.11",
+		"@types/glob": "^7.2.0",
+		"@types/jest": "^26.0.24",
+		"@types/mocha": "^8.2.3",
+		"@types/node": "^14.18.12",
+		"@types/react": "^17.0.40",
+		"@types/react-dom": "^17.0.13",
+		"@types/sinon": "^9.0.11",
+		"@types/uuid": "^8.3.4",
+		"@types/vscode": "^1.65.0",
+		"@typescript-eslint/eslint-plugin": "^4.33.0",
+		"@typescript-eslint/parser": "^4.33.0",
+		"autoprefixer": "^9.8.8",
+		"chai": "^4.3.6",
+		"chai-as-promised": "^7.1.1",
+		"chai-fs": "^2.0.0",
+		"chai-json-schema": "^1.5.1",
+		"chalk": "^4.1.2",
+		"cli-ux": "^5.6.7",
+		"context-map-webpack-plugin": "^0.1.0",
+		"cross-env": "^7.0.3",
+		"css-loader": "^3.6.0",
+		"depcheck": "^1.4.3",
+		"download": "^8.0.0",
+		"enzyme": "^3.11.0",
+		"enzyme-adapter-react-16": "^1.15.6",
+		"eslint": "^6.8.0",
+		"eslint-config-mongodb-js": "^5.0.3",
+		"eslint-plugin-mocha": "^8.2.0",
+		"execa": "^1.0.0",
+		"glob": "^7.2.0",
+		"jest": "^26.6.3",
+		"jest-junit": "^12.3.0",
+		"jest-transform-stub": "^2.0.0",
+		"less": "^3.13.1",
+		"less-loader": "^5.0.0",
+		"meow": "^9.0.0",
+		"mkdirp": "^1.0.4",
+		"mocha": "^8.4.0",
+		"mocha-junit-reporter": "^2.0.2",
+		"mocha-multi": "^1.1.6",
+		"mongodb-ace-autocompleter": "^0.11.0",
+		"mongodb-build-info": "^1.4.0",
+		"mongodb-runner": "^4.8.3",
+		"node-loader": "^0.6.0",
+		"npm-run-all": "^4.1.5",
+		"ora": "^5.4.1",
+		"postcss-loader": "^3.0.0",
+		"pre-commit": "^1.2.2",
+		"semver": "^7.3.5",
+		"sinon": "^9.2.4",
+		"sinon-chai": "^3.7.0",
+		"style-loader": "^1.3.0",
+		"ts-jest": "^26.5.6",
+		"ts-loader": "^8.3.0",
+		"ts-node": "^9.1.1",
+		"typescript": "^4.6.2",
+		"vsce": "^1.103.1",
+		"vscode-test": "^1.6.1",
+		"webpack": "^4.46.0",
+		"webpack-cli": "^3.3.12",
+		"xvfb-maybe": "^0.2.1",
+		"yargs-parser": "^20.2.9"
+	},
+	"precommit": [
+		"check"
+	],
+	"__metadata": {
+		"id": "4b6540da-bfa3-4b88-b9e3-df8f2d161124",
+		"publisherDisplayName": "MongoDB",
+		"publisherId": "96be5892-f133-40ba-88dd-d450cff649f0",
+		"isPreReleaseVersion": false
+	}
```

Comment:
Looks like this file caught some diff from going from 2 space tabs to 4, can we keep it 2 spaces?

Reply:
This is super weird because the package.json shows 2 spaces for me.

<img width="434" alt="Screenshot 2022-08-31 at 17 47 59" src="https://user-images.githubusercontent.com/16307679/187722276-615e90de-3b69-438e-86e7-52002fab6432.png">